deployment/v3/testrig/README.md

```
@@ -1,21 +1,33 @@
+
+## Run UITESTRIG manually
+
+#### CLI
+* Download Kubernetes cluster `kubeconfig` file from `rancher dashboard` to your local.
+* Install `kubectl` package to your local machine.
+* Run UITESTRIG manually via CLI by creating a new job from an existing k8s cronjob.
+  ```
+  kubectl --kubeconfig=<k8s-config-file> -n UITESTRIG create job --from=cronjob/<cronjob-name> <job-name>
+  ```
+  example: 
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)